src/main/java/uk/gov/ons/ctp/integration/rhsvc/service/impl/CaseServiceImpl.java

```diff
@@ -100,6 +104,51 @@ public void fulfilmentRequestByPost(PostalFulfilmentRequestDTO requestBodyDTO)
     createAndSendFulfilments(DeliveryChannel.POST, contact, requestBodyDTO, products, caseDetails);
   }
 
+  @Override
+  public void sendNewCaseEvent(NewCaseDTO newCaseDTO) throws CTPException {
+    log.debug(
+        "Entering createAndSendNewCase",
+        kv("collectionExerciseSid", newCaseDTO.getCollectionExerciseId()),
+        kv("schoolId", newCaseDTO.getSchoolId()),
+        kv("lastName", newCaseDTO.getLastName()));
+
+    NewCasePayloadContent payload = createNewCaseRequestPayload(newCaseDTO);
+
+    eventPublisher.sendEvent(TopicType.NEW_CASE, Source.RESPONDENT_HOME, Channel.RH, payload);
+  }
+
+  private NewCasePayloadContent createNewCaseRequestPayload(NewCaseDTO caseRegistrationDTO)
+      throws CTPException {
+
+    final UUID caseId = UUID.randomUUID();
+    final UUID collectionExerciseId = caseRegistrationDTO.getCollectionExerciseId();
+
+    NewCaseSample sample = new NewCaseSample();
```

Comment:
The normal pattern for these mappings is to use the bean mapper `RHSvcBeanMapper`.

Reply:
This has been changed to use the RHSvcBeanMapper